src/pages/_index.js

```diff
@@ -1,43 +1,80 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-
-import {Button} from 'antd'
-
-import Link from 'components/Link'
+import React, {Component} from 'react'
 
 import Layout from '../components/Layout'
+import ProductCard from '../components/ProductCard'
 
-const IndexPage = ({navigate, location}) => {
-  return (
-    <Layout navigate={navigate} location={location}>
-      <div>
-        <div align="center">
-          <br/>
-          <p style={{color: "cornflowerblue", fontSize: 50, fontWeight: 'bold'}}>
-            Try. Use. Grow.
-          </p>
-          <h4>Всичко от което имаш нужда за твоя бизнес</h4>
-          <br/>
-          <Link to="/politis/get-started/introduction">
-            <Button type="primary" size="large" style={{marginRight: 10}}>Politis</Button>
-          </Link>
+import iamIcon from '../images/iamIcon.png'
+import iamImage from '../images/iamImage.png'
+import fleerpIcon from '../images/fleerpIcon.png'
+import politisIcon from '../images/politisIcon.png'
+import fleerpImage from '../images/fleerpImage.png'
+import politisImage from '../images/politisImage.jpeg'
 
-          <Link to="/fleerp/docs/get-started/tracker-activation">
-            <Button type="primary" size="large" style={{marginRight: 10}}>Fleerp</Button>
-          </Link>
+class IndexPage extends Component {
+    render() {
+        const { navigate, location } = this.props;
 
-          <Link to="/iam/docs/get-started/user-registration">
-            <Button type="primary" size="large" style={{marginRight: 10}}>IAM</Button>
-          </Link>
-        </div>
-      </div>
-    </Layout>
-  )
-}
+        const products = [
+            {
+                icon: politisIcon,
+                image: politisImage,
+                name: 'Politis',
+                url: '/politis/get-started/introduction',
+                title: 'Mobile and Web sales',
+                addition: 'Simple to use',
+                description: 'Politis is powefull cloud based inventory and order management software for easing your business.'
+            },
+            {
+                icon: fleerpIcon,
+                image: fleerpImage,
+                name: 'Fleerp',
+                url: '/fleerp/docs/get-started/tracker-activation',
+                title: 'GPS Tracking System',
+                addition: 'Simple to use',
+                description: 'Fleerp is affordable GPS Tracking System, successfully used by many companies, public sectors and personal households all around the world.'
+            },
+            {
+                icon: iamIcon,
+                image: iamImage,
+                name: 'IAM',
+                url: '/iam/docs/get-started/user-registration',
+                title: 'Mobile and Web sales',
```

Comment:
Identity & Access Management

Reply:
Fine-grained access control and visibility for centrally managing app resources.

Reply:
Ready with changes.